lib/cisco_node_utils/cmd_ref/vxlan_vtep.yaml

```diff
@@ -10,9 +10,28 @@ all_interfaces:
   config_get_token: '/^interface (.*)$/'
 
 host_reachability:
-  config_get_token_append: '/^host-reachability protocol (\S+)/'
-  config_set_append: '<state> host-reachability protocol <proto>'
-  default_value: 'flood'
+  /N(7|9)/:
+    config_get_token_append: '/^host-reachability protocol (\S+)/'
```

Comment:
It is not available on my n3k. Again I'll need your input for 5/6k support expectations.